### ISSUE TITLE
md() function ignored in cols_label()

### ISSUE THREAD

**Issue by ddsjoberg** [rstudio/gt#524]
Hello!

I ran the example in the `cols_label()` help file and it doesn't look like the `md()` function is being honored.  The double stars meant to bold the label are being printed as literal stars.

Congratulations again on the CRAN release!!

``` r
library(gt)
packageVersion("gt")
#> [1] '0.2.0.5'

# Use `countrypops` to create a gt table;
# label columns as before but make them
# bold with markdown formatting
tab_2 <-
  countrypops %>%
  dplyr::select(-contains("code")) %>%
  dplyr::filter(country_name == "Mongolia") %>%
  tail(5) %>%
  gt() %>%
  cols_label(
    country_name = md("**Name**"),
    year = md("**Year**"),
    population = md("**Population**")
  )
```

<sup>Created on 2020-04-02 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>
![image](https://user-images.githubusercontent.com/26774684/78298155-461cda80-74ff-11ea-940c-8eba945e86bf.png)

**Comment by rich-iannone** [comment]
Thanks for reporting this! I'm somehow not seeing what you're seeing though (on my dev system). This is what I got:

<img width="573" alt="countrypops" src="https://user-images.githubusercontent.com/5612024/78363469-cbe56800-7589-11ea-9d87-227c773c9f55.png">

I wonder if the `md()` is the problem or perhaps this is a Windows issue? If you revise the above to use the following code, do you still see the issue?

```r
countrypops %>%
  dplyr::select(-contains("code")) %>%
  dplyr::filter(country_name == "Mongolia") %>%
  tail(5) %>%
  gt() %>%
  cols_label(
    country_name = gt::md("**Name**"),
    year = gt::md("**Year**"),
    population = gt::md("**Population**")
  )
```

In the meantime, we will try to reproduce on our end.

**Comment by ddsjoberg** [comment]
Hopefully it's a me problem!  I ran the code above and saw the same issue. 

Couple of things it may be:
1. Like you said, it may be a Windows issue.  If there is anything I can do to help troubleshoot this, let me know.
1. I've installed dev versions of many tidyverse packages recently preparing gtsummary for the dplyr 1.0 release next month.  Perhaps it's related to the `md()` issue?  I'll do some checking on this on my side and report back
